### DISCUSSION TITLE
Resetprofit.bat interval

### DISCUSSION THREAD

**Discussion by papadrams** [RainbowMiner/RainbowMiner#2041] [Q&A] [answered]
Hallo!
How often does this run? i noticed it affects the accuracy percentage of the price of a coin at any given time.
Is there a way to run it more often? In your experience is it worth to run it more often?

**Comment by RainbowMiner** [comment, marked as answer]
There is no automatic call to this batch file. It's in case somebody wants to reset all pool statistics, thus loosing all collected long time data, from accuracy to week averages. It makes sense to double-click start this script, if there have been large price changes (like a crash) on the cryptomarket. The profit-switcher would normally rebalance within 12-24 hours by itself, but if you get the feeling, it's not switching good enough, you might try to call the Resetprofit.bat.

These are the effects:
- it will delete all pool data from the database
  - all long term data (accuracy, price moving averages) will be lost and restarted
- it will also delete your rig's global profit data (moving averages)
  - if you are using MiningRigRentals, you will have to let the rig run for at least 3 hours, until new rental-rigs will be created or old rental-rigs will be updated

**Reply by papadrams** [reply]
ooooh i was pressing it 3 to 4 tiems a day! Nice
